### DISCUSSION TITLE
Inconsistiencies for administratives stuctures

### DISCUSSION THREAD

**Discussion by tsievert** [osm-search/Nominatim#2946] [Q&A]
Hey, 
 I use an Android app, Badge(r)s, that relies in the background on the administrative structures that Nominatim delivers.

Over time, some inconsistencies or deviations in the hierarchies compared to official resources (e.g. city provided maps, open data etc) have appeared and were fixed by volunteers in the apps own database.

Note: I'm not the dev (that would be @InI4 ) but I have helped on a few occasions to research and fix these inconsistencies.

Is there a way to provide information about these issues so that they would be fixed?
I'm aware of the problems regarding government open data, if it's not published under the correct license, but are there alternative ways to help?

**Comment by lonvia** [comment]
For isolated cases, you could [leave a note](https://www.openstreetmap.org/fixthemap) on openstreetmap.org when you believe that OSM has an error in its administrative boundaries. State the source for what you believe is the correct boundary. Also, please make sure you are logged in when leaving the note so that you get notified if there are questions.

If there are more wide-spread errors where you believe a more global update with government data would be in order, you can ask for advice on https://community.openstreetmap.org. There should be helpful people there to give advice on license compatibility (or how to get one) as well as potential imports.
